rollup-bridge-contracts/.env.example

```diff
@@ -6,8 +6,8 @@ L1_RPC_ENDPOINT="http://localhost:8545"
 WALLET_ADDRESS=""
 
 GETH_RPC_ENDPOINT="http://localhost:8545"
-GETH_PRIVATE_KEY=""
-GETH_WALLET_ADDRESS=""
+GETH_PRIVATE_KEY="002f28996b406c557ff579766af59ba66a3f103b8b90de6e9baad8ae211c0071"
```

Comment:
To not alarm anyone at these values, put a comment here that these are just for test and not in any way related to production.